src/http/sr-modules-operators-keys/sr-modules-operators-keys.controller.ts

```diff
@@ -32,7 +39,36 @@ export class SRModulesOperatorsKeysController {
     description: 'Staking router module_id or contract address.',
   })
   @Get(':module_id/operators/keys')
-  getOperatorsKeys(@Param('module_id') moduleId: ModuleId, @Query() filters: KeyQuery) {
-    return this.srModulesOperatorsKeys.get(moduleId, filters);
+  async getOperatorsKeys(
+    @Param('module_id') moduleId: ModuleId,
+    @Query() filters: KeyQuery,
+    @Res() reply: FastifyReply,
+  ) {
+    await this.entityManager.transactional(
+      async () => {
+        const { operators, keysGenerator, module, meta } = await this.srModulesOperatorsKeys.get(moduleId, filters);
+
+        const jsonStream = JSONStream.stringify(
```

Comment:
This logic with forming response should be placed outside in separate file.
File with stream utils that response generator and generates response